hicexplorer/hicBuildMatrix.py

```diff
@@ -337,21 +337,22 @@ def get_rf_bins(rf_cut_intervals, min_distance=200, max_distance=800):
             continue
 
         # identify overlapping bins
-        if chrom[idx] == chrom[idx-1] and end[idx-1] > start[idx]:
-            middle = start[idx] + int((end[idx-1]-start[idx])/2)
+        if chrom[idx] == chrom[idx - 1] and end[idx - 1] > start[idx]:
+            middle = start[idx] + int((end[idx - 1] - start[idx]) / 2)
             new_start.append(middle)
             new_end.append(middle)
         else:
             new_start.append(start[idx])
-            new_end.append(end[idx-1])
+            new_end.append(end[idx - 1])
         new_chrom.append(chrom[idx])
 
     new_end.append(end[-1])
     assert len(new_chrom) == len(new_start), "error"
     assert len(new_end) == len(new_start), "error"
 
     intervals = zip(new_chrom, new_start, new_end)
-    intervals = [(chrom, start, end) for chrom, start, end in intervals if end - start >= min_distance]
+    intervals = [(chromosome, start_position, end_position) for chromosome, start_position,
```

Comment:
Flake8 liniting complaint about the redefinition of 'start', 'end' and 'chrom' which were already defined in 313.